### ISSUE TITLE
No wheels for Python 3.6 (win32, win_amd64)

### ISSUE THREAD

**Issue by jacekt** [gevent/gevent#928]
* gevent version: 1.2.0
* Python version: 3.6.0
* Operating System: Win7

### Description:

There are no windows wheels for Python 3.6, so `pip install gevent` fails on Win7 (without VC++ Build Tools installed).

### What I've run:

```
$ python -VV
Python 3.6.0 (v3.6.0:41df79263a11, Dec 23 2016, 08:06:12) [MSC v.1900 64 bit (AMD64)]
```

```
$ pip install gevent
Collecting gevent
  Using cached gevent-1.2.0.tar.gz
Requirement already satisfied: greenlet>=0.4.10 in c:\users\user\appdata\local\programs\python\python36\lib\site-packages (from gevent)
Installing collected packages: gevent
  Running setup.py install for gevent: started
    Running setup.py install for gevent: finished with status 'error'
    Complete output from command c:\users\user\appdata\local\programs\python\python36\python.exe -u -c "import setuptools, tokenize;__file__='C:\\cygwin64\\tmp\\pip-build-x3lqd81h\\gevent\\setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record C:\cygwin64\tmp\pip-s93ggw0o-record\install-record.txt --single-version-externally-managed --compile:
    c:\users\user\appdata\local\programs\python\python36\lib\distutils\dist.py:261: UserWarning: Unknown distribution option: 'cffi_modules'
      warnings.warn(msg)
    running install
    running build
    running build_py
    creating build
    creating build\lib.win-amd64-3.6
    creating build\lib.win-amd64-3.6\gevent
    copying src\gevent\backdoor.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\baseserver.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\builtins.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\core.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\event.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\fileobject.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\greenlet.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\hub.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\local.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\lock.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\monkey.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\os.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\pool.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\pywsgi.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\queue.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\resolver_ares.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\resolver_thread.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\select.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\server.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\signal.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\socket.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\ssl.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\subprocess.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\thread.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\threading.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\threadpool.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\timeout.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\util.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\win32util.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\wsgi.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\_compat.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\_fileobjectcommon.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\_fileobjectposix.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\_semaphore.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\_socket2.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\_socket3.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\_socketcommon.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\_ssl2.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\_ssl3.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\_sslgte279.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\_tblib.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\_threading.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\_util.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\_util_py2.py -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\__init__.py -> build\lib.win-amd64-3.6\gevent
    creating build\lib.win-amd64-3.6\gevent\libev
    copying src\gevent\libev\corecffi.py -> build\lib.win-amd64-3.6\gevent\libev
    copying src\gevent\libev\_corecffi_build.py -> build\lib.win-amd64-3.6\gevent\libev
    copying src\gevent\libev\__init__.py -> build\lib.win-amd64-3.6\gevent\libev
    running egg_info
    writing src\gevent.egg-info\PKG-INFO
    writing dependency_links to src\gevent.egg-info\dependency_links.txt
    writing requirements to src\gevent.egg-info\requires.txt
    writing top-level names to src\gevent.egg-info\top_level.txt
    warning: manifest_maker: standard file '-c' not found

    reading manifest file 'src\gevent.egg-info\SOURCES.txt'
    reading manifest template 'MANIFEST.in'
    warning: no files found matching 'known_failures.py'
    warning: no files found matching '.pep8'
    no previously-included directories found matching '*\__pycache__'
    warning: no previously-included files matching '*.so' found anywhere in distribution
    warning: no previously-included files matching '*.o' found anywhere in distribution
    warning: no previously-included files matching '*.lo' found anywhere in distribution
    warning: no previously-included files matching '*.la' found anywhere in distribution
    warning: no previously-included files matching 'config.log' found anywhere in distribution
    warning: no previously-included files matching 'config.status' found anywhere in distribution
    no previously-included directories found matching 'doc\_build'
    warning: no previously-included files matching '*.pyc' found anywhere in distribution
    warning: no previously-included files matching '.coverage' found under directory 'src\greentest'
    no previously-included directories found matching 'src\greentest\htmlcov'
    warning: no previously-included files matching 'stamp-h?' found under directory 'deps\c-ares'
    warning: no previously-included files matching 'ares_build.h.orig' found under directory 'deps\c-ares'
    no previously-included directories found matching 'deps\libev\.deps'
    warning: no previously-included files matching 'Makefile' found under directory 'deps\libev'
    warning: no previously-included files matching 'libtool' found under directory 'deps\libev'
    warning: no previously-included files matching 'stamp-h?' found under directory 'deps\libev'
    warning: no previously-included files matching 'config.h' found under directory 'deps\libev'
    warning: no previously-included files matching '_corecffi.c' found under directory 'src\gevent'
    warning: no previously-included files found matching 'Makefile'
    warning: no previously-included files found matching 'configure-output'
    writing manifest file 'src\gevent.egg-info\SOURCES.txt'
    copying src\gevent\_semaphore.pxd -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\ares.pyx -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\cares.pxd -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\cares_ntop.h -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\cares_pton.h -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\dnshelper.c -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\gevent._semaphore.c -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\gevent.ares.c -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\gevent.ares.h -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\python.pxd -> build\lib.win-amd64-3.6\gevent
    copying src\gevent\libev\_corecffi_cdef.c -> build\lib.win-amd64-3.6\gevent\libev
    copying src\gevent\libev\_corecffi_source.c -> build\lib.win-amd64-3.6\gevent\libev
    copying src\gevent\libev\callbacks.c -> build\lib.win-amd64-3.6\gevent\libev
    copying src\gevent\libev\callbacks.h -> build\lib.win-amd64-3.6\gevent\libev
    copying src\gevent\libev\corecext.ppyx -> build\lib.win-amd64-3.6\gevent\libev
    copying src\gevent\libev\corecext.pyx -> build\lib.win-amd64-3.6\gevent\libev
    copying src\gevent\libev\gevent.corecext.c -> build\lib.win-amd64-3.6\gevent\libev
    copying src\gevent\libev\libev.h -> build\lib.win-amd64-3.6\gevent\libev
    copying src\gevent\libev\libev.pxd -> build\lib.win-amd64-3.6\gevent\libev
    copying src\gevent\libev\libev_vfd.h -> build\lib.win-amd64-3.6\gevent\libev
    copying src\gevent\libev\stathelper.c -> build\lib.win-amd64-3.6\gevent\libev
    running build_ext
    building 'gevent.libev.corecext' extension
    error: Microsoft Visual C++ 14.0 is required. Get it with "Microsoft Visual C++ Build Tools": http://landinghub.visualstudio.com/visual-cpp-build-tools

    ----------------------------------------
Command "c:\users\user\appdata\local\programs\python\python36\python.exe -u -c "import setuptools, tokenize;__file__='C:\\cygwin64\\tmp\\pip-build-x3lqd81h\\gevent\\setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record C:\cygwin64\tmp\pip-s93ggw0o-record\install-record.txt --single-version-externally-managed --compile" failed with error code 1 in C:\cygwin64\tmp\pip-build-x3lqd81h\gevent\
```

**Comment by jamadden** [comment]
Thanks for the report. I'm aware of this. AppVeyor didn't have 3.6.0 final available when 1.2.0 was published. 1.2.1 wheels for 4.6 will be posted. In the meantime you can either install the build tools or snag a 1.2.1a wheel from a recent build at https://ci.appveyor.com/project/denik/gevent/history

**Comment by jacekt** [comment]
Thanks for quick reply.  Got one from appveyor - it works ok.
